python/cudf/cudf/benchmarks/get_datasets.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import getopt
+import sys
+import os
+import shutil
+
+# Update url and dir where datasets needs to be copied
+datasets = {
+    "cuio_dataset": [
+        "https://rapidsai-data.s3.us-east-2.amazonaws.com/cudf/benchmark/"
+        "avro_json_datasets.zip",
+        "cudf/benchmarks/cuio_data/",
+    ],
+}
+
+
+def delete_dir(path):
+    if path == "/" or path == "~":
+        raise ValueError("Trying to delete root/home directory")
+
+    shutil.rmtree(path, ignore_errors=True)
+
+
+def fetch_datasets(urls, dirs):
+    tmp_path = os.getcwd() + "/tmp_benchmark/"
```

Comment:
Should use `os.path.join` here or use `pathlib` (https://docs.python.org/3/library/pathlib.html) instead to handle this more safely.